packages/flutter/lib/src/gestures/long_press.dart

```diff
@@ -133,8 +160,10 @@ class LongPressGestureRecognizer extends PrimaryPointerGestureRecognizer {
   );
 
   bool _longPressAccepted = false;
-
   Offset _longPressOrigin;
+  // The buttons sent by `PointerDownEvent`. If a `PointerMoveEvent` comes with a
+  // different set of buttons, the gesture is rejected and canceled.
```

Comment:
Is it both rejected and canceled?  Or just canceled? I think you probably can just say "is canceled". The rejection is implied.